src/shared/CustomInput/CustomInput.tsx

```diff
@@ -19,6 +20,7 @@ export const CustomInput: FC<Props> = ({
     isInputEnabled,
     inputValue,
     assetValue,
+    balance,
```

Comment:
you are setting default value for `balance` in 2 places in this component. just set it here `balance = '0'`